brainstormforce/ultimate-addons-for-gutenberg
# Fix: Price List - Broken on Responsive devices.

### Description

- https://app.startinfinity.com/b/Mi5zu3VRVzk/5s2b2cSP7a5/ed795069-f12e-41da-85c9-5167b67d5ee2?view=0f0c95d5-d336-4bdd-bb8b-f7cfb212b3d1&t=attributes&redirect=true

### Types of changes




- Bug fix (non-breaking change which fixes an issue)

### How has this been tested?

- Drag and drop Price List in next-release
- Set columns on desktop and tablet to 2 and mobile 1. 
- Check Frontend and editor. You will see issue on mobile device.
- Now Switch to the `price-list-on-reponsive-device` check frontend and editor.

### Checklist:
